docs/wiki/deployment/file-integrity-monitoring.md

```diff
@@ -0,0 +1,72 @@
+As of osquery version 1.4.2, file integrity monitoring support was introduced for linux and darwin variants.  This module reads a list of directories to monitor from the osquery config and details changes and hashes to those selected files in the `file_changes` table.
+
+To get started with FIM (file integrity monitoring), you must first identify which files and directories you wish to monitor. Following the [wildcard rules](../development/wildcard-rules/), you can specify a directory or filename filter to limit the selection of files to monitor.
+
+For example, you may want to monitor `/etc` among other files on a linux system.  After you identify your target files and directories you wish to monitor, add them to a new section in the config `["additional_monitoring"]["file_paths"]`.  
+
+### Example Config
+
+```json
+{
+  "scheduledQueries": [
+    {
+      "name": "crontab",
+      "query": "select * from crontab;",
+      "interval": 300
+    },
+    {
+      "name": "file_changes",
+      "query": "select * from file_changes;",
+      "interval": 300
+    }
```

Comment:
we should update all new docs to use the "new" config schedule syntax: https://osquery.readthedocs.org/en/latest/introduction/using-osqueryd/#configuration-and-query-schedule